CHANGELOG.md

```diff
@@ -54,6 +54,8 @@ All notable changes for each version of this project will be documented in this
         - `indicatorsOrientation` input is added, which can be used to set the position of indicators it can be top or bottom
         - `animationType` input is added, which can be used to set animation when changing slides
         - `indicatorTemplate` directive is added, which can be used to provide a custom indicator for carousel. If this property is not provided, a default indicator template will be used instead.
+        - `nextButtonTemplate` directive is added, which can be used to provide a custom next button. If this property is not provided, a default next button will be used instead.
```

Comment:
Let's make it shorter: "directive is added, which is used to provide a custom next button template. If not provided, a default next button is used.